applications/matter_weather_station/src/app_task.cpp

```diff
@@ -106,7 +106,7 @@ Identify sIdentifyHumidity = { chip::EndpointId{ kHumidityMeasurementEndpointId
 Identify sIdentifyPressure = { chip::EndpointId{ kPressureMeasurementEndpointId }, AppTask::OnIdentifyStart, AppTask::OnIdentifyStop,
 			EMBER_ZCL_IDENTIFY_IDENTIFY_TYPE_AUDIBLE_BEEP };
 
-const device *sBme688SensorDev = device_get_binding(DT_LABEL(DT_INST(0, bosch_bme680)));
+const struct device *sBme688SensorDev = DEVICE_DT_GET_ONE(bosch_bme680);
```

Comment:
Well, it is not idiomatic in C++, rather backwards compatibility we need to live with, so I think it's better not to add it, but I'm OK with merging it, so that you don't have to re-run the CI :).